src/js/modules/infragistics.ui.editors.js

```diff
@@ -11099,6 +11099,13 @@
 			regional = ($.datepicker && typeof reg === "string") ?
 				$.datepicker.regional[ (reg === "defaults" || reg === "en-US") ? "" : reg ] :
 				null;
+

```

Comment:
Please, combine this setting of the "regional" option with the one above (if possible). First get the language, either from the option, util or set default and the get regional setting for that language